com.hma.qlog.desktop

```diff
@@ -0,0 +1,8 @@
+[Desktop Entry]
```

Comment:
It should be upstreamed.

Reply:
sorry, I don't understand .. I don't speak English
I use a translator .. if you can help me
I will thank you

Reply:
He meant that you should send the desktop file into the original (upstream) repository. However, it seems that it is already there: https://github.com/7h0ma5/QLog/blob/master/res/qlog.desktop

Reply:
that is .. it is not necessary to create the .desktop? because the program already has it!
can you correct the code for me? so I realize it and it will serve me for other flatpak

Reply:
Sorry, I am currently too busy with other work stuff. But basically it is just changing this Flatpak manifest line:

`install -D com.hma.qlog.desktop /app/share/applications/com.hma.qlog.desktop`

to

`install -D path/to/com.hma.qlog.desktop /app/share/applications/com.hma.qlog.desktop`

(replace `path/to/` with an actual path inside the tar.gz archive)

Same thing goes for the png/svg icon files etc. If they are included in the upstream tar.gz archive, you can use them from there.

Reply:
Also, don't forget to remove these sources when using the upstream desktop/icon files:

```
      - type: file
        path: com.hma.qlog.png
      - type: file
        path: com.hma.qlog.desktop
```

Reply:
Hm, although I am now not sure whether this is enough.

@barthalion Do Flatpak builder automatically modify the desktop file `Icon=` entry (to match _com.hma.qlog_ instead of _qlog_) if icon/desktop files are installed this way?

Reply:
No, it needs desktop-file-edit.